apps/src/javalab/JavalabEditor.jsx

```diff
@@ -714,6 +714,8 @@ class JavalabEditor extends React.Component {
                     <span>{fileMetadata[tabKey]}</span>
                     {activeTabKey === tabKey && !isReadOnlyWorkspace && (
                       <button
+                        aria-label={i18n.fileOptions()}
+                        aria-expanded={contextTarget === tabKey ? true : false}
```

Comment:
(this logic is copied from ~10 lines below)

Reply:
i think `contextTarget === tabKey` is more readable